src/Microsoft.ML.Data/Transforms/ExtensionsCatalog.cs

```diff
@@ -51,11 +51,15 @@ public InputOutputColumnPair(string outputColumnName, string inputColumnName = n
     public static class TransformExtensionsCatalog
     {
         /// <summary>
-        /// Copies the input column to another column named as specified in <paramref name="outputColumnName"/>.
+        /// Create a <see cref="ColumnCopyingEstimator"/>, which copies the data from the column specified in <paramref name="inputColumnName"/>
+        /// to a new column: <paramref name="outputColumnName"/>.
         /// </summary>
+        /// <remarks>For more information see <see cref="ColumnCopyingEstimator"/>.</remarks>
```

Comment:
>/// <remarks>For more information see <see cref="ColumnCopyingEstimator"/>.</remarks> [](start = 8, length = 85)

we don't have this for trainers. we should be consistent. is this helpful?

Reply:
We do already link to ColumnCopyingEstimator above, but this doesn't hurt.